lib/vdsm/storage/lvm.py

```diff
@@ -265,6 +267,13 @@ def _allowed_devices(devs):
     return devices
 
 
+def _build_devices(devs):
+    if devs:
+        return ",".join(dev for dev in devs)
+    else:
+        return None
```

Comment:
This return None which is never correct. This function accept non empty list of devices, and joins them with ",". There is no need fo function to do this.